sandbox.html

```diff
@@ -611,6 +618,42 @@
   </div>
 </template>
 
+<template data-action="fetchAreaIntersection" data-common="layer">
+  <div>
+    <label for="embeddingTargetId">Embedding target ID *</label>
+    <input name="embeddingTargetId" id="embeddingTargetId" required>
+  </div>
+  <div>
+    <label for="geometry">Intersection Geometry (GeoJSON) *</label>
+    <textarea name="geometry" id="geometry" rows="5" required></textarea>
+    <small>
+      <p>
+        The geometry must result in an area in combination with the buffer specification.
+      </p>
+    </small>
+  </div>
+  <div>
+    <label>
+      <input type="checkbox" name="useMapSrs">
+      Use map SRS
+    </label>
+  </div>
+  <div>
+    <label for="distance">Buffer size</label>
+    <input name="distance" id="distance">
```

Comment:
Proposal: Add type="number" and min="0" here.

Reply:
min 0 is only needed if the intersection geometry is a point or linestring

Reply:
For polygons we also support negative distances!?